### ISSUE TITLE
Processing Sequence of images.

### ISSUE THREAD

**Issue by Dinaa97** [ultralytics/yolov5#8009]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello! I was working lately with yolov5, and I have a task where I want to change the input for yolov5 to be a sequence of images instead of single image for training, for example a tensor containing 5 images along with their labels or something. So, I was wondering if there is a possible way to do that with the current configuration or not? Any information would be helpful.

Thank youuu!

**Comment by glenn-jocher** [comment]
@Dinaa97 you can train with batch size 5:
```
python train.py --batch 5
```

**Comment by Dinaa97** [comment]
Thank you for your reply, but  I am not sure if this would solve the problem, to be more specific I am working on object prediction and tracking from a video, and I have the video as frames, so I want the input to be a stacked frames from one scene for example so that I can continue working with it for predicting the next frame using LSTM. So, I am not quite sure if this would do the same thing.
Please let me know if I am still not clear.

**Comment by glenn-jocher** [comment]
@Dinaa97 👋 Hello! Thanks for asking about object **tracking** in computer vision. YOLOv5 🚀 is an object **detector** that detect, localizes and classifies objects in a single image. It does not connect objects across multiple images, for this you need a tracking solution. A few possible tracking solutions are:

- **Extended Kalman Filter (EKF)**: https://en.wikipedia.org/wiki/Kalman_filter
- **KLT tracker**: https://en.wikipedia.org/wiki/Kanade%E2%80%93Lucas%E2%80%93Tomasi_feature_tracker
- **YOLOv5 DeepSort tracker**: https://github.com/mikel-brostrom/Yolov5_DeepSort_Pytorch

If you looking for a sequential video loading, this does not exist by default in the YOLOv5 repo, but you could build a dataloader for this and then customize a model with this dataloader.

Good luck 🍀 and let us know if you have any other questions!

**Comment by Dinaa97** [comment]
Thank youu for the useful information! I will try them and see what I can achieve.

**Comment by glenn-jocher** [comment]
@Dinaa97 you're welcome! 😊 Good luck with your project, and don't hesitate to reach out if you have any more questions. The YOLO community and our team at Ultralytics are here to help.
